core/domain/exp_jobs_one_off.py

```diff
@@ -666,12 +666,7 @@ def reduce(key, values):
 
 
 class VerifyAllUrlsMatchGcsIdRegexJob(jobs.BaseMapReduceOneOffJobManager):
-    """One-off job for copying the image/audio such that the url instead of
-    {{exp_id}}/assets/image/image.png is
-    exploration/{{exp_id}}/assets/image/image.png. It also involves compressing
-    the images as well as renaming the filenames so that they have dimensions
-    as a part of their name.
-    """
+    """One-off job for verifying the image and audio urls."""
```

Comment:
Actually.... Why did this change? Was the job altered? If not I think having it descriptive as above makes sense.

Reply:
This job is to verify the urls for image and audio but the comment described the job for copying images/audio.

The job was not altered but the comment did not describe what the job does. It is actually same as the comment for this job here: https://github.com/oppia/oppia/blob/3b90f25afcb31a7a08773760b11759bd152c1659/core/domain/exp_jobs_one_off.py#L708

Reply:
Ah I see, thanks for confirming.